llama-index-core/llama_index/core/response_synthesizers/base.py

```diff
@@ -65,6 +65,10 @@ def __init__(
     ) -> None:
         """Init params."""
         self._llm = llm or llm_from_settings_or_context(Settings, service_context)
+
+        if callback_manager:
+            self._llm.callback_manager = callback_manager
```

Comment:
Should this be handled here or should the user have to do this to the LLM before supplying the LLM to base synthesizer?

@logan-markewich ?

Reply:
I think the synthesiser and all its attributes should be using the same callback manager (which is what this aligns)